examples/text-to-speech/run_pipeline.py

```diff
@@ -89,10 +90,20 @@ def main():
     with torch.autocast("hpu", torch.bfloat16, enabled=args.bf16), torch.no_grad(), torch.inference_mode():
         # warm up
         for i in range(args.warmup):
+            if generator.model.config.model_type == "speecht5":
+                # SpeechT5 forces a dropout with training=True, which may zero out some elements randomly.
+                # A random dropout may need different lengths of spectrograms to fit probability thresholds,
+                # which violates the HPU static shape, so we have to fix the seed here.
+                set_seed(555)
```

Comment:
only this seed works or it's just the one working after you tested

Reply:
@libinta Any seed should work. Basically you just need to set the same seed again for each generation.